src/main.rs

```diff
@@ -183,11 +182,18 @@ fn main() -> std::io::Result<()> {
         )
         .get_matches();
 
+    let vals: Vec<&Path>;
+    if let Some(load_paths) = matches.values_of("LOAD_PATH") {
+        vals = load_paths.map(|p| Path::new(p)).collect();
+    } else {
+        vals = Vec::new();
+    }
```

Comment:
Maybe we could do?
```suggestion
    let load_paths = clap::values_t!(matches.values_of("LOAD_PATH"), Path).unwrap_or_default();
```

Reply:
Haha, you tick this but you don't do anything? Why not just apply suggestion or try it out?

Reply:
haha yes sorry I thought I had implemented it but realised this morning I hadn't, so will implement with the other changes I make now :)

Reply:
`Path` does not implement or have a trait for `FromStr` which is required by `values_t` to be able to work so there doesn't seem to be a nice way to do this sadly

Reply:
`PathBuf` could work, it probably does not matter allocating a bit. Opened a discussion at https://github.com/clap-rs/clap/discussions/2013